docs/index.rst

```diff
@@ -14,7 +14,7 @@ Welcome to windrose's documentation!
    :caption: Contents:
 
    Install <install.rst>
-   Usage <usage.rst>
+   Usage <usage-output.ipynb>
```

Comment:
This PR changes how we this part of the documentation. Instead of a rst file we have a notebook that is run at build time, to ensure it is working. That reduces the number of files we need to sync (screenshots, instructions in multiple places, etc) and make it easier for users b/c they can download the notebook and run the docs if they want to follow along.